exercises/perfect-numbers/description.md

```diff
@@ -3,17 +3,17 @@
 Determine if a number is perfect, abundant, or deficient based on Nicomachus' (60 - 120 CE) classification scheme for positive integers.
 
 The Greek mathematician [Nicomachus][nicomachus] devised a classification scheme for positive integers, identifying each as belonging uniquely to the categories of **perfect**, **abundant**, or **deficient** based on their [aliquot sum][aliquot-sum].
-The aliquot sum is defined as the sum of the factors of a number not including the number itself.
+The aliquot sum is defined as the sum of the factors of a `number` not including the `number` itself.
 For example, the aliquot sum of `15` is `1 + 3 + 5 = 9`.
 
-- **Perfect**: aliquot sum = number
-  - 6 is a perfect number because (1 + 2 + 3) = 6
-  - 28 is a perfect number because (1 + 2 + 4 + 7 + 14) = 28
-- **Abundant**: aliquot sum > number
-  - 12 is an abundant number because (1 + 2 + 3 + 4 + 6) = 16
-  - 24 is an abundant number because (1 + 2 + 3 + 4 + 6 + 8 + 12) = 36
-- **Deficient**: aliquot sum < number
-  - 8 is a deficient number because (1 + 2 + 4) = 7
+- **Perfect**: when a `number` equals its `aliquot sum`
```

Comment:
Should `number` and `aliquot sum` be quoted rather than code blocked?

Reply:
I will take that as a no, then. ;)

Reply:
It's perhaps easier to read wth the code blocks marking them as variables which also draws us to their earlier mention -- but I don't mind. It currently looks like

* * * *

The `aliquot sum` is defined as the sum of the factors of a `number` not including the `number` itself.
For example, the `aliquot sum` of `15` is `1 + 3 + 5 = 9`.

- **Perfect**: when a `number` equals its `aliquot sum`
- **Abundant**: when a `number` is less than its `aliquot sum`
- **Deficient**: when a `number` is greater than its `aliquot sum`

* * * *

Reply:
I thought we were describing the idea, rather than actual code.  It seems that `aloquot sum` may or may not be an invocation of a function or method in some languages.

I definitely will not stand in the way of it though.

It could look like:

---

The "aliquot sum" is defined as the sum of factors of a `number not including the `number` itself.
For example, the "aliquot sum" of `15` is `1 + 3 + 5` or `9`.


  - **Perfect**: when `number` equals its "aliquot sum"
  - **Abundant**: when `number` less than its "aliquot sum"
  - **Deficient**: when `number` greater than its "aliquot sum"

---

Reply:
The _aliquot sum_ is defined as the sum of factors of a `number` not including the `number` itself. For example, the _aliquot sum_ of `15` is `1 + 3 + 5` or `9`.

 * **Perfect**: when `number` equals its _aliquot sum_
 * **Abundant**: when `number` less than its _aliquot sum_
 * **Deficient**: when `number` greater than its _aliquot sum_

—-

Italics to me emphasize the technical term aliquot sum but also distinct from other terms that are bolded, a different category of emphasis. Quote marks to me don’t convey that much emphasis compared to italics.

Reply:
Why would aliquot sum be quoted in any way? It might make sense to bold it when defining it, but once defined it should be used as normal prose.

Reply:
I would also prefer bolding it

Reply:
I've turned them into links referencing the relevant heading.

Reply:
Everyone happy with this ?